File: KotlinCode/src/leetcode/easy/JewelsInStones.kt
package leetcode.easy

/**
 * Leetcode 771. Jewels and Stones
 * https://leetcode.com/problems/jewels-and-stones/description/
 */
class JewelsInStones{
    fun numJewelsInStones(J: String, S: String): Int {
        var count:Int = 0
        for (chJ in J) {
            for (chS in S) {
                if (chJ == chS) {
                    count++
                }
            }
        }
        return count
    }
}

fun main(args: Array<String>) {
    val test = JewelsInStones()
    println(test.numJewelsInStones("aA", "aAAbbbb"))
    println(test.numJewelsInStones("z", "ZZ"))
}

File: PythonCode/leetcode/easy/JewelsInStones.py
#!/usr/bin/python3
# -*- coding: utf-8 -*-

# @Time    : 2018/2/10 9:26
# @Author  : bearyang
# @Site    : 
# @File    : JewelsInStones.py
# @Software: PyCharm


class Solution:
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        count = 0
        for chJ in J:
            for chS in S:
                if chJ == chS:
                    count += 1
        return count


test = Solution()
print(test.numJewelsInStones("aA", "aAAbbbb"))
print(test.numJewelsInStones("z", "ZZ"))